docs/managed-mode.md

```diff
@@ -20,6 +20,9 @@ automations:
       - action: code-review@v1
         args:
           guidelines: {{ loadReviewGuidelines() | dump }}
+
+is:
+  bot_author: {{ pr.author | match(list=[\"github-actions\", \"_bot_\", \"[bot]\", \"dependabot\"]) | some }}
```

Comment:
can you remove the escaping in all exmaples please (they come becase i took it from logs)

```suggestion
is:
  bot_author: {{ pr.author | match(list=["github-actions", "_bot_", "[bot]", "dependabot"]) | some }}
```